### PR TITLE
Allow transformers-0.6 and mtl-2.3

### PR DESCRIPTION
**Pull request by sjakobi**
https://hackage.haskell.org/package/transformers-0.6.0.0/changelog
https://hackage.haskell.org/package/mtl-2.3/changelog

---

Blocked on:

* [x] https://github.com/haskell/mtl/issues/88 (needs [release](https://github.com/haskell/mtl/issues/86))

### CODE REVIEW

**Review by Gabriel439 — APPROVED** [review]
I added you as a maintainer so that you can merge this when it's unblocked

**Comment by sjakobi** [comment]
I have used my Hackage trustee powers to create a matching Hackage revision: https://hackage.haskell.org/package/mmorph-1.2.0/revisions/

**Pull request merged**
